pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -685,6 +686,9 @@ func (lrw *logicalReplicationWriterProcessor) flushBuffer(
 			chunkEnd++
 		}
 		chunk := todo[0:chunkEnd]
+		if len(chunk) > chunkSize*2 {
+			lrw.debug.LogOversizedChunk()
```

Comment:
i was originally thinking this would be an actual log line, not something in the vtable.

Reply:
also, i dont think this var gets printed in the vtable currently, if you do want to keep in the vtable.

Reply:
This was enough to see it in stack inspection tools. In practice the chunks todo/done have been more useful.

Reply:
I backed this change out completely in the end.